webui-src/app/mail/mail_util.js

```diff
@@ -208,7 +208,7 @@ const MessageView = () => {
                 m('button[id=show-less][style="display: none;"]', {
                   onclick: () => {
                     document.querySelector('#show-more').style.display = 'block';
-                    document.querySelector('#truncate').style.height = '1rem';
+                    document.querySelector('#truncate').style.height = '22px';
                     document.querySelector('#show-less').style.display = 'none';
```

Comment:
Isn't a size in pixels prone to look very differently on high DPI screens?

Reply:
Nope, It will adjust according to the UI, I tested it on small and big screen.
Also, 1 rem = 16px, so rem also does the same thing there.

Reply:
rem is actually equal to the font size of it's root, although it won't make a difference in this case.

Reply:
yes, since we didn't set it, by default it will be 16px.